pkg/gce-pd-csi-driver/server.go

```diff
@@ -92,6 +93,13 @@ func (s *nonBlockingGRPCServer) serve(endpoint string, ids csi.IdentityServer, c
 		klog.Fatalf("%v endpoint scheme not supported", u.Scheme)
 	}
 
+	if u.Scheme == "unix" {
+		listenDir, _ := filepath.Split(addr)
+		if _, err := os.Stat(listenDir); err != nil {
```

Comment:
would prefer a check for an `os.IsNotExist(err)` explicitly and the error message thrown could be more illuminating such as: "Expected Kubelet plugin watcher to create parent dir *dirname* but did not find such a di"